next-jdbc/src/ragtime/next_jdbc.clj

```diff
@@ -30,8 +35,16 @@
     (get-table-metadata* (:connectable datasource))
 
     :else
-    (with-open [conn (jdbc/get-connection datasource)]
-      (get-table-metadata* conn))))
+    (with-open [c (jdbc/get-connection datasource)]
+      (get-table-metadata* c))))
+
+(defn- unquote-table-name [q table-name]
+  (if q
+    ;; Remove db quote around the schema and table name; ex.: "\"schema\".\"table\""
+    (let [pattern (re-pattern (format "^%s|%s(?=\\.)|(?<=\\.)%s|%s$"
+                                q q q q))]
```

Comment:
We probably want to quote `q` here, and `quote-char` would be a more descriptive name. So something more like:

```clojure
(if quote-char
  (let [re (->> (Pattern/quote quote-char)
                (format "^%1$s|%1$s(?=\\.)|(?<=\\.)%1$s|%1$s$")
                (re-pattern))]
    (str/replace table-name re ""))
  table-name)
```

Reply:
Wonderful idea! And thank you vermy much for teaching me about `1$`!